Documentation/gettingstarted/dns.rst

```diff
@@ -0,0 +1,203 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************************************
+Getting Started Using DNS-Based Policies
+****************************************
+
+This document serves as an introduction for using Cilium to enforce DNS-based
+security policies for Kubernetes pods.
+
+.. include:: gsg_intro.rst
+.. include:: minikube_intro.rst
+.. include:: cilium_install.rst
+
+Step 2: Deploy the Demo Application
+===================================
+
+DNS-based policies are very useful for controlling access to services running outside the Kubernetes cluster. Whether it is cloud services such as S3, DynamoDB, RDS, etc. or it is services in your corporate environment running outside the cluster, DNS provides reliably identity for these external services. CIDR or IP-based policies are cumbersome and hard to maintain as the IPs associated with external services can change frequently. The Cilium DNS-based policies provide an easy mechanism to specify access control while leaving the harder aspects of resolving and enforcing IP-based filters to Cilium. 
+
+In this guide we will learn about:
+
+- Controlling egress access to services outside the cluster using DNS-based policies
+- Using patterns (or wildcards) to whitelist a subset of DNS domains
+- Combining DNS, port and API-level rules for restricting external service access
+
+Step 3: Create Example App
+==========================
+
+The file ``dns-sw-app.yaml`` contains ``deathstar``, an internal cluster service and two pods, ``mediabot`` and ``spaceship``. The ``mediabot`` pod needs access to twitter API services for managing the empire's tweets. However, the ``spaceship`` pods shouldn't have any external service access for obvious reasons. Both the pods should be able to access the internal ``deathstar`` service. 
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/dns-sw-app.yaml
+    $ kubectl get po,svc
+    NAME                             READY   STATUS    RESTARTS   AGE
+    pod/deathstar-5fc7c7795d-7ljr8   1/1     Running   0          14s
+    pod/deathstar-5fc7c7795d-lvw69   1/1     Running   0          14s
+    pod/mediabot                     1/1     Running   0          14s
+    pod/spaceship                    1/1     Running   0          14s
+
+    NAME                 TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    service/deathstar    ClusterIP   10.110.127.146   <none>        80/TCP    14s
+    service/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   16m
+
+Step 4: Check Current Access
```

Comment:
I think this step is too verbose. It adds a lot of reading time without contributing much value to the guide. We don't have to prove over and over again that a lack of policy means default allow and thus free access. We can document it once if you want and refer to it.